import.sh

```diff
@@ -8,6 +8,11 @@ find . -type l -not -path './.git/*' -exec rm {} \;
 curl -sL https://github.com/jemalloc/jemalloc/releases/download/4.4.0/jemalloc-4.4.0.tar.bz2 | tar jxf - -C internal --strip-components=1
 patch -p1 -d internal < secure_getenv.patch
 
+# symlink so cgo compiles them
```

Comment:
The comment about `git clean` was borrowed from c-rocksdb's import.sh, and this is the style followed by that file - just trying to make the two more consistent.